### ISSUE TITLE
[Bug] Infinite initialization loop in NX workspace

### ISSUE THREAD

**Issue by czemar** [ng-doc/ng-doc#67]
### Is this a regression?

I don't know

### Description

After manual installation in empty nx integrated workspace I am experiencing an infinite initialization loop. 

### Please provide the exception or error you saw

```true
> nx run docs:serve:development

⠼ NgDoc: Initializing...
```

### OS

Windows

### Browser

None

### Node version

v18.16.0

### Anything else?

Same behavior with `docs:build:production`

Versions:
```
"@angular/core": "~16.0.0",
"@ng-doc/app": "^16.6.1",
"@ng-doc/builder": "^16.6.1",
"@ng-doc/core": "^16.6.1",
"@ng-doc/ui-kit": "^16.6.1",
"@nx/angular": "16.3.2",
"nx": "16.3.2",
"parsimmon": "^1.18.1",
```

**Comment by skoropadas** [comment]
Hey, maybe I forgot to mention something in manual steps, I need to check it, but have you tried to use `npm install @ng-doc/add && npx nx g @ng-doc/add:ng-add` for NX? This command should work fine for NX workspace

**Comment by skoropadas** [comment]
Ok, this is regression, just create at least one page and app should be intialized correctly, I'll fix it soon

**Comment by czemar** [comment]
Thanks a lot! I tried a few times and managed to get it working. This library seems great!

Your advice was helpful. I also noticed a small mistake in the documentation: [link](https://ng-doc.com/docs/getting-started/installation#importing-global-modules)
- There is an import for `NgDocUiKitRootModule` that doesn't seem to be used.
- There are two missing modules: `NgDocNavbarModule` and `NgDocSidebarModule`.

But after looking at the demo I found how to fix my other issues.
